app/models/user.rb

```diff
@@ -9,4 +9,9 @@ class User < ApplicationRecord
   has_many :posts
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
+
+  has_many :friendships
+
+  has_many :accepted_friendships, -> { where(status: true) }, class_name: 'Friendship'
+  has_many :pending_friendships, -> { where(status: false) }, class_name: 'Friendship'
```

Comment:
This is a good start for the next milestone but before this, you should look into adding an inverted friendship relationship. See this link on how it works and improves the performance of your application: https://smartfunnycool.com/friendships-in-activerecord/